addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureScanRule.java

```diff
@@ -801,10 +775,21 @@ private void findBackupFile(HttpMessage originalMessage) throws Exception {
                     "Some error occurred when looking for a backup file for '{}'",
                     originalMessage.getRequestHeader().getURI(),
                     e);
-            return;
         }
     }
 
+    private HttpMessage setMessageCookies(HttpMessage newMsg, HttpMessage originalMsg) {
```

Comment:
`private static void`

Reply:
Void?

Reply:
The method does not need to return anything, the message passed in is changed either way.